tasks/store-publish/publishUi.ts

```diff
@@ -60,6 +63,11 @@ function gatherParams()
 
     taskParams.packages = packages.map(p => p.trim()).filter(p => p.length != 0);
 
+    if (taskParams.deletePackages)
+    {
+        taskParams.numberOfPackagesToKeep = parseInt(tl.getInput('numberOfPackagesToKeep', true));
```

Comment:
I suggest to make this input editable, therefore we should check the value is smaller than MAX_PACKAGES_PER_GROUP